e2etest/e2etest_suite_test.go

```diff
@@ -70,3 +83,218 @@ func TestE2E(t *testing.T) {
 	gomega.RegisterFailHandler(framework.Fail)
 	ginkgo.RunSpecsWithDefaultAndCustomReporters(t, "E2E Suite", r)
 }
+
+var _ = ginkgo.BeforeSuite(func() {
+	cs, err := framework.LoadClientset()
+	framework.ExpectNoError(err)
+	_, err = cs.CoreV1().ConfigMaps(testNameSpace).Get(context.TODO(), "config", metav1.GetOptions{})
+	if errors.IsNotFound(err) {
+		err = configureMetalLB(cs)
+		framework.ExpectNoError(err)
+	} else if err == nil {
```

Comment:
Maybe we should just fail if it has already been configured? That seems like an error condition.